<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSCGenerator_EditEnvironmentDockWidget</class>
 <widget class="QDockWidget" name="OSCGenerator_EditEnvironmentDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Environment</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="AddEnvironment_Button">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>342</width>
         <height>404</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="DateTimeGroup">
          <property name="title">
           <string>Time of Day</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="TimeAnimation">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Animation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="TimeofDay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="dateTime">
              <datetime>
               <hour>6</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2020</year>
               <month>10</month>
               <day>23</day>
              </datetime>
             </property>
             <property name="date">
              <date>
               <year>2020</year>
               <month>10</month>
               <day>23</day>
              </date>
             </property>
             <property name="time">
              <time>
               <hour>6</hour>
               <minute>0</minute>
               <second>0</second>
              </time>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-ddThh:mm:ss</string>
             </property>
             <property name="timeSpec">
              <enum>Qt::UTC</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="WeatherGroup">
          <property name="title">
           <string>Weather</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="CloudState_Label">
             <property name="text">
              <string>Cloud State</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="Fog_VisualRange_Label">
             <property name="text">
              <string>Fog Visual Range</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="CloudState">
             <item>
              <property name="text">
               <string>free</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cloudy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>overcast</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rainy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>skyOff</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Fog_VisualRange">
             <property name="text">
              <string>100000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="Sun_Intensity_Label">
             <property name="text">
              <string>Intensity</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="Sun_Azimuth_Label">
             <property name="text">
              <string>Azimuth</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="Sun_Elevation_Label">
             <property name="text">
              <string>Elevation</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="Sun_Intensity">
             <property name="text">
              <string>0.85</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="Sun_Azimuth">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="Sun_Elevation">
             <property name="text">
              <string>1.31</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="Precip_Intensity">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="Precip_Type_Label">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="Precip_Intensity_Label">
             <property name="text">
              <string>Intensity</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="Precipitation_Label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Precipitation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="Sun_Label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Sun&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="Precip_Type">
             <item>
              <property name="text">
               <string>dry</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rain</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>snow</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
